section3/exercises/loops-arrays.js

```diff
@@ -19,32 +19,53 @@ for (var i = 0; i < animals.length; i++) {
 // YOU DO: Write code below that iterates through a list of animals and prints "The
 // <animal> is awesome!" for each animal.
```

Comment:
Beautiful work in this file - your style and syntax makes this code REALLY easy to read, which your colleagues will appreciate!